model/database/cosmosdb-metrics.yaml

```diff
@@ -0,0 +1,57 @@
+groups:
+  - id: metric.db.cosmosdb.operation.request_charge
+    type: metric
+    metric_name: db.cosmosdb.operation.request_charge
+    brief: "Request charge consumed by the operation."
+    instrument: histogram
+    unit: "{request_units}"
+    stability: experimental
+    extends: attributes.db.cosmosdb.minimal
+
+  - id: metric.db.cosmosdb.operation.regions_contacted
+    type: metric
+    metric_name: db.cosmosdb.operation.regions_contacted
+    brief: "Number of regions contacted for the operation"
+    instrument: histogram
+    unit: "{count}"
+    stability: experimental
+    extends: attributes.db.cosmosdb.minimal
+
+  - id: metric.db.cosmosdb.client.active_instances
+    type: metric
+    metric_name: db.cosmosdb.client.active_instances
+    brief: "Number of active client instances"
+    instrument: histogram
+    unit: "{count}"
+    stability: experimental
+    extends: attributes.db.cosmosdb.minimal
+
+  - id: attributes.db.cosmosdb.minimal
+    type: attribute_group
+    brief: 'Azure Cosmos DB attributes'
+    attributes:
+      - ref: db.system
+        requirement_level:
+          conditionally_required: if available
+      - ref: db.collection.name
+        requirement_level:
+          conditionally_required: if available
+      - ref: db.namespace
+        requirement_level:
+          conditionally_required: If available.
+      - ref: db.operation.name
+        requirement_level: required
+      - ref: server.address
```

Comment:
please add `server.port` with something like `conditionally_required: if not default (443)` in requirement level.